perfdash/Dockerfile

```diff
@@ -12,8 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM debian:jessie
-RUN apt-get update
-RUN apt-get install -y -qq ca-certificates
```

Comment:
Does distroless contain ca-certificates out of box? I can't see any line installing this in the new code?

Reply:
It's integrated with the image base : https://github.com/GoogleContainerTools/distroless/blob/70f4a32ab305eec38d9d1c6e5bce2e3a9b92f877/base/base.bzl#L14-L16.